### PR TITLE
GatewayAccount digitalWallets property

### PR DESCRIPTION
**Pull request by adamaltman**
Alternative to #51

### CODE REVIEW

File: openapi/components/schemas/PaymentCards/DigitalWallets.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+type: object
+description: Configure which digital wallets are enabled.
+title: Digital Wallets
+properties:
+  Apple Pay:
```
Comment (creocoder) [inline openapi/components/schemas/PaymentCards/DigitalWallets.yaml:5]: Probably would better to rename this to `applePay`?

File: openapi/components/schemas/PaymentCards/DigitalWallets.yaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    type: object
+    description: The Apple Pay digital wallet configuration. Skip if not using Apple Pay.
+    required:
+        - isEnabled
+    properties:
+      isEnabled:
+        type: boolean
+        description: Set true if Apple Pay is enabled.
+  Google Pay:
```
Comment (creocoder) [inline openapi/components/schemas/PaymentCards/DigitalWallets.yaml:14]: Probably would better to rename this to `googlePay`?

File: openapi/components/schemas/PaymentCards/DigitalWallets.yaml (hunk 1 above)
Comment (adamaltman) [reply openapi/components/schemas/PaymentCards/DigitalWallets.yaml:5]: It is possible.

**Review by creocoder — DISMISSED** [review]
LGTM

**Review by colinbird — DISMISSED** [review]

**Review by mclassic — DISMISSED** [review]

File: openapi/components/schemas/PaymentCards/DigitalWallets.yaml [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    required:
+        - isEnabled
+    properties:
+      isEnabled:
+        type: boolean
+        description: Set true if Apple Pay is enabled.
+        default: false
+  googlePay:
+    type: object
+    description: The Google Pay digital wallet configuration. Skip if not using Google Pay.
```
Comment (colinbird) [inline openapi/components/schemas/PaymentCards/DigitalWallets.yaml:17]: TM here too?
Maybe just here? the brand guidelines state:

> Ensure the first mention of Google Pay on any feature page or developer documentation includes the trademark symbol, ™ , on the first or most prominent time it appears.

Comment (iamdevice) [reply openapi/components/schemas/PaymentCards/DigitalWallets.yaml:17]: You are right. Fixed

**Review by subenson — APPROVED** [review]

**Review by colinbird — APPROVED** [review]

**Pull request merged**